selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithTerminalTest.java

```diff
@@ -156,13 +158,18 @@ public void shouldAppearMCDialogs() {
   public void shouldScrollIntoTerminal() throws InterruptedException {
     openMC("/");
 
-    // check scrolling of the terminal
-    terminal.movePageDownListTerminal("opt");
-    terminal.moveDownListTerminal(".dockerenv");
-    terminal.waitExpectedTextIntoTerminal(".dockerenv");
-    terminal.movePageUpListTerminal("projects");
-    terminal.moveUpListTerminal("bin");
-    terminal.waitExpectedTextIntoTerminal("bin");
+    try {
+      // check scrolling of the terminal
+      terminal.movePageDownListTerminal("opt");
+      terminal.moveDownListTerminal(".dockerenv");
+      terminal.waitExpectedTextIntoTerminal(".dockerenv");
+      terminal.movePageUpListTerminal("projects");
+      terminal.moveUpListTerminal("bin");
+      terminal.waitExpectedTextIntoTerminal("bin");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7592");
```

Comment:
Does it mean that any line in `try` block may cause an issue?

Reply:
These lines are in try/catch block because in the terminal after buttons clicking, like movePageDownListTerminal(), some content can be missed.
![user_permission-13](https://user-images.githubusercontent.com/7760565/33319725-52121b70-d448-11e7-9db2-799621f7199e.gif)

Reply:
If we are tacking about che6 https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/96/testReport/junit/org.eclipse.che.selenium.dashboard/WorkspaceDetailsTest/workingWithProjects/, there is exact command which throws an exception. It's better to place only one command to try-catch to localize the error place precisely.

Reply:
Another point is to use overloaded version of _fail()_ method to save information about initial exception:
`fail("Known issue https://github.com/eclipse/che/issues/7592", ex);`